addons/website_membership/controllers/main.py

```diff
@@ -46,41 +43,38 @@ def members(self, membership_id=None, country_name=None, country_id=0, page=1, *
         ]
         if membership_id and membership_id != 'free':
             membership_id = int(membership_id)
-            base_line_domain.append(('membership_id', '=', membership_id))
-            membership = product_obj.browse(cr, uid, membership_id, context=context)
-        else:
-            membership = None
+            base_line_domain += [
+                ('membership_id', '=', membership_id), ('date_to', '>=', today),
+                ('date_from', '<', today), ('state', '=', 'paid')]
```

Comment:
already base_line_domain contain default this domain - [('date_to', '>=', today), ('date_from', '<', today), ('state', '=', 'paid')] . is this required to add again ?